scripts/document.sh

```diff
@@ -49,7 +49,8 @@ jazzy \
     --documentation="docs/guides/*.md" \
     --root-url "${BASE_URL}/directions/${RELEASE_VERSION}/" \
     --theme ${THEME} \
-    --output ${OUTPUT}
+    --output ${OUTPUT} \
+    --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO
```

Comment:
When attempting to test this script locally, I kept receiving the error: 

```Running xcodebuild
Could not successfully run `xcodebuild`.
Please check the build arguments.
```

After looking into this issue in the jazzy repo, adding these build parameters helped me build the docs successfully. Please advise if there's a better way to handle.